### PR TITLE
Fixed test failing on centos6

### PR DESCRIPTION
**Pull request by Annu149**
Step 'An entry to {action} {env} env var is added on all hosts of cluster' is failing on centos6 platform. This step was added as part of https://github.com/greenplum-db/gpdb/pull/15504

Reason:
systemctl command is being used to restart sshd service which is not present on centos6 so it is failing to restart sshd.

Fix:
Updated the step to check for the os platform and version. If it is a centos os wth version 6 then use 'sudo service sshd restart' command to restart sshd service.

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: gpMgmt/test/behave/mgmt_utils/steps/unreachable_hosts_mgmt_utils.py [hunk 1]
```diff
@@ -110,6 +112,13 @@ def impl(context, action, env):
 
     cmds = [cmdstr, "sudo systemctl restart sshd.service"]
 
+    # if it is a centos6 machine, use service command to restart sshd
+    x = platform.linux_distribution()
```
Comment (jnihal) [inline gpMgmt/test/behave/mgmt_utils/steps/unreachable_hosts_mgmt_utils.py:116]: Would be better to unpack the values here.
```
os, version, _ = platform.linux_distribution()
```

File: gpMgmt/test/behave/mgmt_utils/steps/unreachable_hosts_mgmt_utils.py [hunk 2]
```diff
@@ -110,6 +112,13 @@ def impl(context, action, env):
...
+    name = x[0].lower()
+    version = x[1]
+    if 'centos' in name and '6' in version[:1]:
```
Comment (jnihal) [inline gpMgmt/test/behave/mgmt_utils/steps/unreachable_hosts_mgmt_utils.py:119]: The assertion for the version should be more deterministic using `==`.

File: gpMgmt/test/behave/mgmt_utils/steps/unreachable_hosts_mgmt_utils.py [hunk 3]
```diff
@@ -110,6 +112,12 @@ def impl(context, action, env):
 
     cmds = [cmdstr, "sudo systemctl restart sshd.service"]
```
Comment (shrakesh) [inline gpMgmt/test/behave/mgmt_utils/steps/unreachable_hosts_mgmt_utils.py:113]: this can be added in the else part of the centos 6 check.

**Review by shrakesh — APPROVED** [review]
approved with minor comments.

**Review by jnihal — APPROVED** [review]

**Review by piyushc01 — APPROVED** [review]

**Pull request merged**